packages/angular/src/generators/application/schema.json

```diff
@@ -158,6 +158,12 @@
       "default": false,
       "hidden": true,
       "x-priority": "internal"
+    },
+    "skipTsConfig": {
```

Comment:
I don't think this should be an option. Angular applications sort of NEED to create the tsconfig right?

Would you actually get a working application if you skipped generating the tsConfig?

Reply:
done. remove this flag